src/main.go

```diff
@@ -33,12 +33,150 @@ func getOlPath(ctx *cli.Context) (string, error) {
 	return filepath.Abs(olPath)
 }
 
+func getOlWorkerProc(ctx *cli.Context) (int, error) {
```

Comment:
I have a generic question about the generic environment / configuration of open lambda. 

**`ol` command only work in the "right" directory**
I think we assume open lambda worker will be a singleton in the system, and therefore both the `default-ol` directory and config files should be fixed at some mount point. But the command itself is set to be a relative path, and only take effect if the sys admin is in the correct directory. Why is this the case?